docs/platforms/python/configuration/options.mdx

```diff
@@ -29,7 +29,7 @@ Learn more about [DSN utilization](/product/sentry-basics/dsn-explainer/#dsn-uti
 
 Environment variable: `SENTRY_DEBUG`
 
-Turns debug mode on or off. If debug is enabled SDK will attempt to print out useful debugging information if something goes wrong with sending the event. The default is always `false`. It's generally not recommended to turn it on in production, though turning `debug` mode on will not cause any safety concerns.
+Turns debug mode on or off. If `debug` is enabled, the SDK will attempt to print out debugging information. This can be useful for example if something goes wrong with event sending. The default is always `False`. It's generally not recommended to turn it on in production because of the increase in log output.
```

Comment:
```suggestion
Turns debug mode on or off. When `True`, the SDK will attempt to print out debugging information. This can be useful if something goes wrong with event sending. The default is always `False`. It's generally not recommended to turn it on in production because of the increase in log output.
```